pointerevents/pointerevent_attributes_hoverable_pointers-manual.html

```diff
@@ -9,6 +9,17 @@
         <!-- Additional helper script for common checks across event types -->
         <script type="text/javascript" src="pointerevent_support.js"></script>
         <script>
+            function driver() {
+            }
+            driver.prototype.sendPointerActions = function(actions) {
```

Comment:
Obviously this bit wouldn't be in WPT itself but some implementation details of the browser.  For now maybe move it out to a "automation-polyfill.js" or something to make that more clear?